grandnode/grandnode
# #422 add possibility to publish / unpublish newsletter categories

Resolves #422   
Type: **feature**

## Issue
Newsletter categories can't be unpublished.

## Solution
Add possibility to unpublish newsletter categories.

## Testing
1. Promotions -> Newsletter -> Newsletter category.
2. Add a newsletter category.
3. tick / untick "Published"
4. Go to frontend, enter E-Mail in the newsletter section and click the envelop
5. If "Published" was ticked a popup should show with the newsletter category
6. If "Published" was unticked and no other newsletter category is published no popup should show
7. If "Published" was unticked and another newsletter category is published the popup should show without the newsletter category were "Publish" was unticked
